modules/nf-core/subread/featurecounts/main.nf

```diff
@@ -11,9 +11,9 @@ process SUBREAD_FEATURECOUNTS {
     tuple val(meta), path(bams), path(annotation)
 
     output:
-    tuple val(meta), path("*featureCounts.txt")        , emit: counts
-    tuple val(meta), path("*featureCounts.txt.summary"), emit: summary
-    path "versions.yml"                                , emit: versions
+    tuple val(meta), path("*featureCounts.txt")                 , emit: counts
+    tuple val(meta), path("*featureCounts.txt.summary")         , emit: summary
+    path "versions.yml"                                         , emit: versions
```

Comment:
Also, try to be consistent with the use of parentheses while defining your input